packages/dd-trace/src/appsec/reporter.js

```diff
@@ -209,6 +236,14 @@ function finishRequest (req, res) {
     rootSpan.setTag('_dd.appsec.rasp.rule.eval', metrics.raspEvalCount)
   }
 
+  if (metrics?.raspTimeouts) {
+    rootSpan.setTag('_dd.appsec.rasp.timeout', metrics.raspTimeouts)
+  }
+
+  if (metrics?.raspErrorCode) {
+    rootSpan.setTag('_dd.appsec.rasp.error', metrics.raspErrorCode)
+  }
```

Comment:
tiny nit, but this could be just above `_dd.appsec.rasp.rule.eval`, so we can have the same order for both waf and rasp